Source/DOM classes/Unported or Partial DOM/SVGLineElement.m

```diff
@@ -41,6 +52,10 @@ -(CALayer *)newLayer
 	CGPathAddLineToPoint(path, NULL, _x2, _y2);
 	
 	CALayer* result = [SVGHelperUtilities newCALayerForPathBasedSVGElement:self withPath:path];
+	if (CGRectIsEmpty(result.bounds)) // empty rects don't get rendered
```

Comment:
This is a bug in Apple's [CALayer bounds], right?

Because a CGPath of a horizontal or vertical line (and Apple knows this!) should not have an empty bounds. My understanding (Apple hasn't documented it properly): the bounds is meant to be the "rendered" bounds, i.e. including the fact that lines have 0.5 pixesl width (at least in CA they do: by default they are 1 pixel wide, with half the line rendered either side in pixels).

Presumably, we are also affected by the same bug in SVGRectElement, SVGPolylineElement, etx?